thomas-bryson/wk01/2-tues/strings/js/strings.js

```diff
@@ -0,0 +1,54 @@
+const DrEvil = function(amount) {
+    if (amount !== 10000000) {
+    const amountDollars = (' ' + amount + ' dollars')
+    return amountDollars;
+    } else;
+    const amountDollarsPinky = (' ' + amount + ' dollars (pinky)')
+    return amountDollarsPinky
+}
```

Comment:
This function only really needs one if statement. 
```suggestion
const DrEvil = function(amount) {
    let output =  amount + ' dollars';
    
    if (amount === 1000000) {
       output = output + ' (pinky)';
    }
    
    return amountDollarsPinky
}
```